mapswipe_workers/scripts/archive_old_projects.py

```diff
@@ -0,0 +1,69 @@
+from mapswipe_workers import auth
+from mapswipe_workers.definitions import logger
+
+
+def get_old_projects():
+    """
+    Get all projects from Firebase which have been created
+    before we switched to v2.
+    """
+    fb_db = auth.firebaseDB()
+    ref = fb_db.reference("projects")
+    projects = ref.get()
+    logger.info("got old projects from firebase")
+    return projects
+
+
+def move_project_data_to_v2(project_id, project_data):
+    """
+    Copy project information from old path to v2/projects in Firebase.
+    Add status=archived attribute.
+    """
+    project_data["status"] = "archived"
+    fb_db = auth.firebaseDB()
+    fb_db.reference("v2/projects/{0}".format(project_id)).set(project_data)
+    fb_db.reference("projects/{0}".format(project_id)).set({})
```

Comment:
If previous statement errors this statement could still be executed and delete all old projects and therefor we loos all old projects.
Either we wrap this in a transaction or delete old projects when we are sure the transfer was successful.

Reply:
I used a firebase transaction function